src/Compilers/Core/Portable/Syntax/SyntaxNode.cs

```diff
@@ -932,7 +933,8 @@ internal static SyntaxTrivia FindTriviaByOffset(SyntaxNode node, int textOffset,
                     {
                         if (element.IsNode)
                         {
-                            return FindTriviaByOffset(element.AsNode(), textOffset, stepInto);
```

Comment:
Good catch. I don't see any reason why not. 
I'll fix those too and add VB test.